src/transaction-path/parse-path.js

```diff
@@ -0,0 +1,54 @@
+const { ESCAPE_CHAR, DELIMITER, MAX_PARTS } = require('./constants');
+
+// Stupid JavaScript doesn't support regexp's lookbehind
+// This hack is for simulating regexp positive and negative lookbehind
+// If the negative lookbehind worked regex[] would be pretty simple:
+//
+//   new RegExp(DELIMTIER + "(?<!\" + ESCAPE_CHAR + DELIMITER+")")
+//
+//   e.g: /:(?<!\\:)/g
+//
+// Taken from:
+//   http://blog.stevenlevithan.com/archives/javascript-regex-lookbehind
+//
+// Reference:
+//   http://stackoverflow.com/questions/13993793/error-using-both-lookahead-and-look-behind-regex
+//
+// Gist:
+//   https://gist.github.com/slevithan/2387872
+//
```

Comment:
dtto `/** */` ... unfortunately the note about stupid JavaScript cannot be deleted in this case, it seems to be still relevant.